app/questionnaire/placeholder_transforms.py

```diff
@@ -41,9 +44,21 @@ def __init__(
 
     input_date_format = "%Y-%m-%d"
 
-    def format_currency(self, number: Decimal | float, currency: str = "GBP") -> str:
-        formatted_currency: str = format_currency(number, currency, locale=self.locale)
-        return formatted_currency
+    def format_currency(
+        self,
+        number: Decimal | float,
+        currency: str = "GBP",
+        decimal_limit: int | None = None,
+    ) -> str:
+        if number or number == 0:
```

Comment:
again, I think either the typing should be fixed and this should be `if number is not None` or the check deleted

Reply:
I can't see a scenario where it would be valid to pass None. The schema should only call format if it knows it has something to format. it is Author's responsibility to ensure that is handled via schema logic. If it got None and we rendered an empty string, it wouldn't throw an error in prod but is ultimately hiding a flawed schema. So if when removing this, we see errors in prod, it would mean schemas aren't guarding this scenario and was incorrectly passing.

Reply:
Updated now assuming that None's are not valid 👍

Reply:
Worth checking if there are any places that call this, where answer could genuinely be None (shouldn't be any if schema logic is right?)

Reply:
I've scanned through the business schemas checking the `format_currency` and `format_number` methods (don't think any use `format_unit` at the moment) and it seems all the conditional `format_currency` transforms are using mandatory questions and the ones that aren't have default `0` values so think we are ok there.

For `format_number` I found instances in the `1_0112` and `1_0215` Retail Sales schemas where there is a conditional question asking for the number of full-time employees , which is then being used in a transform. This does now fall over when not answered given the above changes. All other instances i found seemed to be mandatory, only found these instances but possible I may have missed some.

Reply:
Just checking before resolving @berroar - was there a follow up for these two somewhere?